shared/utilities/getTicketStatus/test.js

```diff
@@ -0,0 +1,130 @@
+/* eslint max-len: ["error", 2000] */
```

Comment:
Fix the formatting instead of violating the style. No point in having a linter if we just ignore it.